rest_witchcraft/viewsets.py

```diff
@@ -26,6 +26,7 @@ class ModelViewSet(
     mixins.UpdateModelMixin,
     DestroyModelMixin,
     mixins.ListModelMixin,
+    ExpandableQuerySerializerMixin,
```

Comment:
Don't wanna include this in the default `ModelViewSet`, this should be as close as possible to drf's model viewset. Lets create another viewset, call it `ExpandableModelViewSet`